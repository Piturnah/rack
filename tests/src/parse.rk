fn main in
  0 while 1 + dup 11 < do
    "we are parsing\n" puts  
  end
end
