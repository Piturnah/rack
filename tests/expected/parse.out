----STDOUT----
we are parsing
we are parsing
we are parsing
we are parsing
we are parsing
we are parsing
we are parsing
we are parsing
we are parsing
we are parsing

----STDERR----
